tools/ci_import

```diff
@@ -0,0 +1,99 @@
+#!/bin/env python
+
+import argparse
+import json
+import os
+import sys
+import zipfile
+import tempfile
+from os import path
+
+import gitlab
+
+IMI_FOLDER = "generated-image-infos"
+MANIFEST_DB_PROJECT_ID = 36844106
+
+
+def import_image_infos(pipeline, token, verbose=False):
```

Comment:
I would be in favor of splitting this method into smaller ones, it's on the verge of being hard to read. One suggestion that I made in a later comment is to factor out `load_imis_from_gitlab_job` method.